examples/app.py

```diff
@@ -35,35 +51,93 @@
 
 You can find the database in `examples/app.db`.
 
-Run the development server:
+
+3. Transform the file `authority.xml` from MARC21 to a json file:
 
 .. code-block:: console
 
-    $ flask -a app.py run -p 5000 -h '0.0.0.0'
+    $ dojson do -i authority.xml -l marcxml marc21 > records.json
+
+
+4. Load the file in the application importing the json file:
+
+.. code-block:: console
+
+    $ flask -a app.py records create records.json
+
+
+5. Create a root collection (By default, the first collection is the root
```

Comment:
`Create a root collection using the CLI. This collection will become the root of new tree by default.`

--
please free to re-phrase it better